server/sonar-server/src/test/java/org/sonar/server/measure/index/ProjectMeasuresIndexerTest.java

```diff
@@ -125,6 +126,7 @@ public void update_index_when_project_is_created() {
     assertThat(result.getSuccess()).isEqualTo(1L);
   }
 
+  @Ignore
   @Test
   public void update_index_when_project_tags_are_updated() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)